packages/apps/networks/42.json

```diff
@@ -1,17 +1,22 @@
 [
   {
     "contractName": "HighRollerApp",
-    "address": "0xb98d753127f2c9C4f60FB7ED54d6cf082b29f7FD",
-    "transactionHash": "0x139919c4824dbb42848e190a9f753bfe1d878101697b8a7371921f69f0c467f5"
+    "address": "0x2a88a838A36f7700C44329790dED3B6249c76C07",
```

Comment:
How come these addresses changed?

Reply:
I just ran `yarn migrate`, I'm not sure why it migrated all the contracts not just my contract. Reverted the addresses and hashes.